app/views/ladder/components/Leaderboard.vue

```diff
@@ -254,7 +254,15 @@
 
 <template lang="pug">
   .table-responsive(:class="{'col-md-6': scoreType=='arena'}")
-    div(id="histogram-display-humans", class="histogram-display", data-team-name='humans' v-if="scoreType == 'arena'")
+    div(v-if="scoreType == 'arena'")
+      div(:class="{hide: !showContactUs}" id="contact-us-info")
```

Comment:
where is this showContactUs variable coming from?

Reply:
oh if you look at `leaderboard.coffee`, that file passed in `dataObj` which is this vue's data ()  😂  
our tricky vue solution with backbone 😂